module1/homework/lengthSort/sort.cpp

```diff
@@ -0,0 +1,21 @@
+#include "sort.hpp"
+#include <algorithm>
+
+std::deque<std::string> lengthSort(std::forward_list<std::string>& list) {
+    list.sort();
+    std::deque<std::string> dequeSorted;
+
+    for (auto &el : list) {
+        dequeSorted.emplace_back(el);
+    }
+
```

Comment:
You can initialize deque like this:
std::deque<std::string> dequeSorted{list.begin(), list.end()};
No need for push_back for

Reply:
+1